mergify_engine/actions/copy.py

```diff
@@ -55,6 +55,7 @@ def get_config_schema(
             voluptuous.Required("branches", default=[]): [str],
             voluptuous.Required("regexes", default=[]): [voluptuous.Coerce(Regex)],
             voluptuous.Required("ignore_conflicts", default=True): bool,
+            voluptuous.Required("append_original_description", default=False): bool,
```

Comment:
I got the idea, but to make it more generic like the title, maybe the new attribute should just be:

```
voluptuous.Required("body_suffix", default=""): types.Jinja2)
```

And the configuration will look like:

```
backport:
    body_suffix: "{{ body }}"
```

WDYT @jd ?

Reply:
What about having the whole body being customizable?
Where you could reuse part of the original pull request like `{{ body }}` in it and maybe some other data such as `{{ git_cherry_pick_output }}` etc?

Reply:
We can do that too, `rendering_template()` already support additional variables